challenges/dog-photo-gallery/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Dog photo gallery</title>
+    <script src="./script.js" defer></script>
```

Comment:
Your HTML file is well-structured and includes the necessary elements. Good job including the script with the defer attribute to ensure it loads after the HTML.

Reply:
Thank you!